phpcs.xml

```diff
@@ -13,8 +15,17 @@
 	<exclude-pattern>*/node_modules/*</exclude-pattern>
 	<exclude-pattern>*/vendor/*</exclude-pattern>
 
+	<!-- Show progress, show the error codes for each message (source). -->
+	<arg value="ps"/>
+
+	<!-- Strip the filepaths down to the relevant bit. -->
+	<arg name="basepath" value="./"/>
```

Comment:
The comment from it doesn't sound descriptive as like in the PHPCS command line `--help`:

> A path to strip from the front of file paths inside reports

Reply:
I have pushed a commit making the comment more clear.